src/OrbitGl/TimeGraph.cpp

```diff
@@ -380,40 +380,50 @@ void TimeGraph::ProcessValueTrackingTimer(const TimerInfo& timer_info) {
 }
 
 void TimeGraph::ProcessMemoryTrackingTimer(const TimerInfo& timer_info) {
-  const std::string kTotalLabel = "Memory Total (kB)";
+  const std::string kTotalLabel = "Memory Total";
   const std::string kUnusedLabel = "Memory Unused (kB)";
   const std::string kBuffersOrCachedLabel = "Memory Buffers / Cached (kB)";
   const std::string kUsedLabel = "Memory Used (kB)";
+  const std::string kWarningThresholdLabel = "Production Limit";
 
-  std::optional<std::pair<std::string, uint64_t>> warning_threshold_label_and_kb = std::nullopt;
+  std::optional<std::pair<std::string, uint64_t>> warning_threshold_pretty_name_and_raw_value =
+      std::nullopt;
+  const uint64_t kKilobytesToBytes = 1024;
   if (app_->GetCollectMemoryInfo()) {
-    warning_threshold_label_and_kb =
-        std::make_pair("Production Limit (kB)", app_->GetMemoryWarningThresholdKb());
+    uint64_t warning_threshold_kb = app_->GetMemoryWarningThresholdKb();
+    std::string pretty_size = GetPrettySize(warning_threshold_kb * kKilobytesToBytes);
+    std::string pretty_name = absl::StrFormat("%s: %s", kWarningThresholdLabel, pretty_size);
+    warning_threshold_pretty_name_and_raw_value = std::make_pair(pretty_name, warning_threshold_kb);
   }
 
   int64_t total_kb = orbit_api::Decode<int64_t>(timer_info.registers(0));
-  std::optional<std::pair<std::string, int64_t>> total_label_and_kb = std::nullopt;
-  if (total_kb != kMissingInfo) total_label_and_kb = std::make_pair(kTotalLabel, total_kb);
+  std::optional<std::pair<std::string, int64_t>> total_pretty_name_and_raw_value = std::nullopt;
+  if (total_kb != kMissingInfo) {
+    std::string pretty_size = GetPrettySize(total_kb * kKilobytesToBytes);
+    std::string pretty_name = absl::StrFormat("%s: %s", kTotalLabel, pretty_size);
+    total_pretty_name_and_raw_value = std::make_pair(pretty_name, total_kb);
```

Comment:
This variable name doesn't say me anything. I won't comment a suggestion because I know that you are modifying code here.

Reply:
mmm, if you still want to use the optional in the Set method, I would rename the variables:
pretty_total_size
pretty_total_label
pretty_total_label_and_value

Reply:
Removed optional arguments in the Set methods and also renamed the variables. Thanks!